resources/metadata/gitopsUpdateDeployment.yaml

```diff
@@ -0,0 +1,112 @@
+metadata:
+  name: gitopsUpdateDeployment
+  description: Updates Kubernetes Deployment Manifest in Infrastructure Git Repo
+  longDescription: |
+    This step allows you to update the deployment manifest in another repository.
```

Comment:
[In the docs](https://sap.github.io/jenkins-library/steps/fortifyExecuteScan/) `description` and `longDescription` are rendered directly after each other. So the `longDescription` should kind of extend the `description`. But `description` is also used as command description standalone in the `CLI`.